fieldservice_route/models/fsm_route_dayroute.py

```diff
@@ -34,7 +39,9 @@ class FSMRouteDayRoute(models.Model):
     last_location_id = fields.Many2one(
         comodel_name="fsm.location", string="Last Location"
     )
-    date_start_planned = fields.Datetime(string="Planned Start Time")
+    date_start_planned = fields.Datetime(
+        string="Planned Start Time", compute="_compute_date_start_planned", store=True
```

Comment:
This should be `readonly=False` for having a computed writable field and editable like before.